spec/views/idv/socure/errors/show.html.erb_spec.rb

```diff
@@ -0,0 +1,130 @@
+require 'rails_helper'
+
+RSpec.describe 'idv/socure/errors/show.html.erb' do
+  include Devise::Test::ControllerHelpers
+
+  let(:remaining_submit_attempts) { 5 }
+  let(:in_person_url) { nil }
+  let(:flow_path) { :standard }
+  let(:sp) { create(:service_provider) }
+  let(:decorated_sp_session) do
+    ServiceProviderSession.new(
+      sp:,
+      view_context: nil,
+      sp_session: nil,
+      service_provider_request: nil,
+    )
+  end
+
+  context 'timeout error' do
+    let(:error_code) { :timeout }
+    let(:presenter) do
+      SocureErrorPresenter.new(
+        error_code:,
+        remaining_attempts: remaining_submit_attempts,
+        sp_name: decorated_sp_session&.sp_name || APP_NAME,
+        issuer: decorated_sp_session&.sp_issuer,
+        flow_path:,
+      )
+    end
```

Comment:
If we define error_code on the outer most scope we can define presenter once. Then it is clearer in the context what is actually relevant.

```suggestion
  let(:error_code) { nil }
  let(:decorated_sp_session) do
    ServiceProviderSession.new(
      sp:,
      view_context: nil,
      sp_session: nil,
      service_provider_request: nil,
    )
  end
  let(:presenter) do
    SocureErrorPresenter.new(
      error_code:,
      remaining_attempts: remaining_submit_attempts,
      sp_name: decorated_sp_session&.sp_name || APP_NAME,
      issuer: decorated_sp_session&.sp_issuer,
      flow_path:,
    )
  end

  context 'timeout error' do
    let(:error_code) { :timeout }
```